app/components/wallet/WalletTrezorDialog.js

```diff
@@ -237,13 +237,11 @@ const ProgressStateOption = {
 
 type TrezorDeviceInfo = {
   valid: boolean;
-  error: ?{
-    id: string,
-    defaultMessage: string,
-    description: string
-  },
-  cardanoGetPublicKeyResult: CardanoGetPublicKey, // Trezor device CardanoGetPublicKey object
-  features: Features // Trezor device CardanoGetPublicKey object
+  error: ?LocalizableError,
+  // Trezor device CardanoGetPublicKey object
+  cardanoGetPublicKeyResult: CardanoGetPublicKey,
```

Comment:
maybe for name we could use:

`cardanoPublicKey`

and for type:

`TrezorCardanoPublicKey`